Assets/Mirror/Tests/Editor/NetworkWriterTest.cs

```diff
@@ -523,8 +521,8 @@ public void TestToArray()
             // set position back by one
             writer.Position = 1;
 
-            // Changing the position should not alter the size of the data
-            Assert.That(writer.ToArray().Length, Is.EqualTo(2));
+            // Changing the position should alter the size of the data
+            Assert.That(writer.ToArray().Length, Is.EqualTo(1));
```

Comment:
clearly failing this test.   I want all the data in the NtworkWriter,  not all the data up to position
there is no guarantee that position and length are the same,  that is just how we happen to use it.